AUTHORS

```diff
@@ -1,123 +1,5 @@
-This AUTHORS file is generated by ./script/authors.sh
 Thank you to everyone whom has contributed to streamlink / livestreamer.
```

Comment:
Not your edit, I know, but isn't "whom" grammatically incorrect here? "Who" is the subject of that clause.

Reply:
so it should be who instead of whom ?

Reply:
I agree `who` sounds better grammatically.